src/main/java/org/apache/commons/configuration2/INIConfiguration.java

```diff
@@ -349,7 +364,7 @@ public void setCommentLeadingCharsUsedInInput(final String separator) {
      *
      * @param writer - The writer to save the configuration to.
      * @throws ConfigurationException If an error occurs while writing the configuration
-     * @throws IOException if an I/O error occurs.
+     * @throws IOException            if an I/O error occurs.
```

Comment:
Do not change the formatting please.

Reply:
Reverted unnecessary formatting changes.  
Sorry for the inconvenience.